edx_proctoring/templates/proctoring/seq_proctored_practice_exam_entrance.html

```diff
@@ -0,0 +1,70 @@
+{% load i18n %}
+<div class="sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Would you like to take {{ display_name }} as a practice proctored exam?
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Since this course is offered for credit, there will be one or more proctored exams in this course. Before you encounter a proctored exam that will count towards your grade and credit eligibility. You now have the option to take a practice proctored exam. Online proctoring is one requirement towards being eligible for credit.
+    {% endblocktrans %}
+  </p>
+  <p>
+    {% blocktrans %}
+      This practice proctored exam is being provided to you so that you can learn more about proctoring. Also this
+      practice exam will give you a chance to test your computer system with the proctoring software without it
+      counting towards your grade.
+    {% endblocktrans %}
+  </p>
+  <p>
+    {% blocktrans %}
+        While it is not requireed to take this practice proctored exam, it is highly recommended.
+    {% endblocktrans %}
+  </p>
+  <div class="gated-sequence">
+    <span><i class="fa fa-lock"></i></span>
+    <a class="start-timed-exam" data-ajax-url="{{enter_exam_endpoint}}" data-exam-id="{{exam_id}}" data-attempt-proctored=true data-start-immediately=false>
+      {% trans "Yes, I want to take this practice exam with online proctoring" %}
+    </a>
+    <p>
+      {% blocktrans %}
+        You will be guided through installing {{platform_name}} approved online proctoring software and
```

Comment:
"You will be guided through steps to set up online proctoring software and to perform various check. When you start the exam, the timer on the right shows the time remaining in the exam."